src/org/daisy/dotify/formatter/impl/obfl/ObflParserImpl.java

```diff
@@ -874,6 +875,8 @@ void parseBlock(
         FormatterCore fc,
         TextProperties tp
     ) throws XMLStreamException {
+        externalReferenceObject = null;
```

Comment:
If you want to empty the map, please use `externalReferenceObject.clear();`

Reply:
I don't want an empty map. I want to remove the object.

Reply:
Just to clarify, this could be implemented as an object. This is not a map. Only my implementation uses a map other implementations might choose other data types.